appengine-java21/static-files/src/main/webapp/index.html

```diff
@@ -1,6 +1,5 @@
-<?xml version="1.0" encoding="utf-8"?>
 <!--
- Copyright 2017 Google Inc.
+Copyright 2024 Google LLC
```

Comment:
nit: the year should stay at 2017.

Reply:
Fixed, as well as all the old dates I could see.